packages/store/test/web3.test.js

```diff
@@ -17,30 +17,17 @@ describe('Resolving Web3', () => {
   describe('with customProvider', () => {
     beforeAll(async () => {
       global.window = {}
-      web3Options = { web3: { customProvider: global.provider } }
+      web3Options = { customProvider: global.provider }
     })
 
     test('get web3', async () => {
-      gen = initializeWeb3({ options: web3Options })
+      gen = initializeWeb3(web3Options)
 
       // First action dispatched
       expect(gen.next().value).toEqual(put({ type: Action.WEB3_INITIALIZED }))
 
       resolvedWeb3 = gen.next().value
-
-      hasWeb3Shape(resolvedWeb3)
-    })
-
-    test('get network ID', async () => {
```

Comment:
@cds-amal just wondering why this test was removed.

Reply:
Checking the network id is an arbitrary test for a custom provider.